.azuredevops/deploy-platform-bootstrap-env.yaml

```diff
@@ -67,6 +67,13 @@ extends:
           - 'refs/heads/main'
         azureRegions:
           primary: 'UKSouth'
+      - name: 'dev1'
+        outputTemplateChange: Skip
+        serviceConnection: AZR-ADP-SSV5
```

Comment:
this needs to be environment level service connection, which we need to create

Reply:
but as it's bootstrap... i think its OK?

Reply:
We'll need to refactor this pipeline to clean up some of the environment and tenant specific resources. But for now it's working fine.